make_printable_es.sh

```diff
@@ -0,0 +1 @@
+po4a --no-update po4a.cfg && latexmk -shell-escape -pdf main_es
```

Comment:
This is not the purpose of this script. Please make an alias on your local machine and delete this file from the repo.